src/config/matomo-config.ts

```
@@ -8,4 +8,6 @@ export class MatomoConfig implements Config {
   public hostUrl: string;
 
   public siteId: string;
+
+  public dimensionId: Number;
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)